frontend/src/store/questions/constants.js

```diff
@@ -0,0 +1,23 @@
+import {
+  getQuestions,
+  getQuestion,
+  getAnswers
+} from '@/utils/api/questions'
+
+export const ACTIONS = {
+  GET_FEED: Symbol('getQuestionFeed'),
+  GET_QUESTION: Symbol('getQuestion'),
+  GET_ANSWERS: Symbol('getAnswers')
+}
+
+export const REQUEST = {
+  [ACTIONS.GET_FEED]: getQuestions,
```

Comment:
what do these square brackets do?

Reply:
Allows the string to be dynamically set as the object key.

Reply:
neato